src/chains/ethereum/console.log/tests/index.test.ts

```diff
@@ -34,7 +34,10 @@ describe("@ganache/console.log", () => {
       wallet: { deterministic: true, totalAccounts: 1 },
       miner: { blockGasLimit: "0xfffffffff" },
       logging: { logger },
-      chain: { allowUnlimitedContractSize: true },
+      chain: {
+        allowUnlimitedContractSize: true,
+        allowUnlimitedInitCodeSize: true
```

Comment:
The console.log test contract is so big we need to lift the initcode limit as well as the contract size limit.